site/ja/blog/new-in-devtools-94/index.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: "layouts/blog-post.njk"
+title: "DevTools の新機能 (Chrome 94)"
+authors:
+  - jecelynyeen
+date: 2021-08-24
+updated: 2021-08-24
+description:
+  "好みの言語での DevTools の利用, 新しい Nest Hub デバイス, 新しい CSS コンテナクエリバッチなど。"
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/js1uKEulNBGj0PoJkqZq.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-94
+---
+
+*Thanks [yoichiro](https://github.com/yoichiro) for the translations, [technohippy](https://github.com/technohippy) and [lacolaco](https://github.com/lacolaco) for the reviews!*
```

Comment:
I think this line can be translated to Japanese as well!

Reply:
@chybie I translated this sentence into Japanese at 91aae57.